### PR TITLE
Add Toolbar Icons for Zoom In, Zoom Out and Reset.

### PR DESCRIPTION
**Pull request by muddana-satish**
depends on #https://github.com/theia-ide/timeline-chart/pull/140

Signed-off-by: muddana-satish <satish.muddana@ericsson.com>

### CODE REVIEW

**Comment by muddana-satish** [comment]
Added three icons for zoom in, zoom out and reset. 

**Zoom In/ Zoom Out:** Zooms at the center of the chart if the selection range is not specified. Else zooms at the middle of the selection range.

**Reset:** Scales to the default level.


Kapture 2021-06-14 at 13 46 44

**Comment by MatthewKhouzam** [comment]
I would like Stefan or Erica's opinion about the icons. I instinctively thougth the + or - was to add or remove entries.

**Comment by MatthewKhouzam** [comment]
Also reset is backwards reload. ;)

File: theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Command } from '@theia/core';
+
+export namespace TraceViewerToolbarCommands {
+
+    export const ZOOM_IN: Command = {
+        id: 'trace.viewer.zoomin',
+        label: 'Zoom In',
+        iconClass: 'fa fa-plus',
```
Comment (MatthewKhouzam) [inline theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts:8]: https://fontawesome.com/v4.7/icon/search-plus ?

File: theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    };
+
+    export const ZOOM_OUT: Command = {
+        id: 'trace.viewer.toolbar.zoomout',
+        label: 'Zoom Out',
+        iconClass: 'fa fa-minus',
```
Comment (MatthewKhouzam) [inline theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts:14]: fa-search-minus ?

File: theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    };
+
+    export const RESET: Command = {
+        id: 'trace.viewer.toolbar.reset',
+        label: 'Reset',
+        iconClass: 'fa fa-undo',
```
Comment (MatthewKhouzam) [inline theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts:20]: https://fontawesome.com/v4.7/icon/map

**Review by MatthewKhouzam — COMMENTED** [review]
Not a hard minus, I have questions and suggestions though for the icon selection

**Comment by bhufmann** [comment]
I think the idea was to use: 
image

**Comment by MatthewKhouzam** [comment]
@bhufmann  yes, the box makes a world of difference IMO. It gives a context that the windows is zoomed.

File: theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts (hunk 1 above)
Comment (MatthewKhouzam) [reply theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts:8]: https://fontawesome.com/v4.7/icon/plus-square-o !!! this one! this one!

File: theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts (hunk 2 above)
Comment (MatthewKhouzam) [reply theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts:14]: https://fontawesome.com/v4.7/icon/minus-square-o

File: theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts (hunk 1 above)
Comment (muddana-satish) [reply theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts:8]: image
 
They're very dull for some reason :/

**Comment by ebugden** [comment]
> **bhufmann:** I think the idea was to use:
> image

Without reading this exchange in detail, my understanding is that the icons added in this PR should be the exact icons used in the wireframe that @ssmagula tested on users recently. One of the questions asked was what the icons meant (without having a chance to try the buttons) and users had no issues identifying the actions associated with them. Other than that I haven't been much involved in the icon selection process.

**Comment by MatthewKhouzam** [comment]
I found two of the three icons... see the comments. 3rd one is tricky.

**Comment by ebugden** [comment]
Is it not in the Figma project? I don't think the icons were created by hand.

**Review by MatthewKhouzam — APPROVED** [review]
I am OK with this, we need to do two things:

1. Update the home button when we find the right icon
2. make it work on the time axis in the future so it will affect line charts too.

**Review by bhufmann — APPROVED** [review]
Looks good to me. Thanks!

**Comment by ebugden** [comment]
> I am OK with this, we need to do two things:
> 
> 1. Update the home button when we find the right icon
> 2. make it work on the time axis in the future so it will affect line charts too.

Can these Todos be made into issues, so they don't get forgotten?

**Pull request merged**